DASH-MPD-PATCH-RESTRICTED.xsd

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE schema [
+ <!ENTITY ncname "\i\c*">
+ <!ENTITY qname  "(&ncname;:)?&ncname;">
+ <!ENTITY aname  "@&qname;">
+ <!ENTITY pos    "\[\d+\]">
+ <!ENTITY attr   "\[&aname;='(.)*'\]|\[&aname;=&quot;(.)*&quot;\]">
+ <!ENTITY cond   "&attr;|&pos;">
+ <!ENTITY step   "(&qname;|\*)(&cond;)*">
+ <!ENTITY nspa   "namespace::&ncname;">
+ <!ENTITY last   "(&step;|&aname;|&nspa;)">
```

Comment:
Entity descriptions taken directly from the [patch operations xsd](https://www.iana.org/assignments/xml-registry/schema/patch-ops.xsd) with the following changes:

| Entity Name | Change | Reason |
|------------------|------------|------------|
| `&value;`/`&valueq` | Removed | Matching based on child element node value not permitted by matching guidelines
| `&cond;` | Changed | Removed option of `&value;` due to the removal of that entity
| `&pi;`/`&piq;` | Removed | Matching based on processing instructions not permitted by matching guidelines
| `&id;` | Removed | Matching based on unique document id location not permitted by matching guidelines
| `&com;` | Removed | Comments not a permitted matching structure
| `&text;` | Removed | Node text not a permitted matching structure
| `&cnodes;` | Removed | All sub-entities eliminated
| `&child;` | Removed | With removal of `&cnodes;` just a simple alias for `&step;`
| `&last;` | Changed | Replaced `&child;` with `&step;` to collapse entity alias

These changes are made based on my understanding of the restrictions as we defined them in AMD1, open to a different interpretation discussion.